doc/concepts/tutorials.rst

```diff
@@ -0,0 +1,52 @@
+Tutorials
+=========
+
+Tutorials are one of the most useful contributions you can make because they are the first thing many new users see.
+We have separated the sections of this website based on `The Documentation System <https://documentation.divio.com/>`_.
+Tutorials are **Learning-Oriented** and have these qualities:
+
+* learning by doing
+* inspiring confidence
+* repeatability
+* immediate sense of achievement
+* concretness, not abstraction
+* minimum necessary explanation
+* no distractions
+* no options or decisions
+
+In the end, the reader should feel a sense of accomplishment.
+For that, they have to take some actions that they were instructed to.
```

Comment:
I sort of understand what this sentence is getting at, but it doesn't feel complete. 

```suggestion
For that to happen, they need to play an active role throughout the tutorial.
```